packages/react-query/src/shared/hooks/types.ts

```diff
@@ -47,9 +48,11 @@ export interface TRPCUseQueryBaseOptions {
   trpc?: TRPCReactRequestOptions;
 }
 
-export interface UseTRPCQueryOptions<TPath, TInput, TOutput, TData, TError>
-  extends UseQueryOptions<TOutput, TError, TData, [TPath, TInput]>,
-    TRPCUseQueryBaseOptions {}
+export interface UseTRPCQueryOptions<TPath, TInput, TOutput, TData, TError, TQueryData>
+  extends Omit<UseBaseQueryOptions<TOutput, TError, TData, TQueryData, [TPath, TInput]>, 'placeholderData'>,
+  TRPCUseQueryBaseOptions {
+    placeholderData?: TOutput | PlaceholderDataFunction<TOutput>;
```

Comment:
I tried `TOutput`, `TData` and `TQueryData` here, seems to yield the same behavior (or not work at all, can't remember which)